vaadin-grid-flow-parent/vaadin-grid-flow/src/test/java/com/vaadin/flow/component/grid/AbstractGridMultiSelectionModelSelectedItemsTest.java

```diff
@@ -0,0 +1,119 @@
+/*
```

Comment:
As you has this amazing test case, test setup, it would be good to check whats happening if:
- we deselect all the item?
- select one or two items multiple times,
- deselect one or two items multiple times,
and maybe some more I can not think of.

This is really cool, thank you for this!